src/components/ReviewsPage.jsx

```diff
@@ -0,0 +1,29 @@
+import {useState, useEffect} from "react";
+import { fetchReviews } from "../utils";
+import ReviewsList from "./ReviewsList";
+
+const ReviewsPage = () => {
+const [reviews, setReviews] = useState([]);
+
+useEffect(() => {
```

Comment:
Could we have some state to track the status of the asynchronous fetch? Would be useful to use this to return some feedback to the user.